src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -27,6 +27,7 @@ public static class EnvironmentSettingNames
         public const string AppInsightsConnectionString = "APPLICATIONINSIGHTS_CONNECTION_STRING";
         public const string AppInsightsQuickPulseAuthApiKey = "APPINSIGHTS_QUICKPULSEAUTHAPIKEY";
         public const string FunctionsExtensionVersion = "FUNCTIONS_EXTENSION_VERSION";
+        public const string FunctionWorkerRuntime = "FUNCTIONS_WORKER_RUNTIME";
```

Comment:
This is already defined here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs#L10

Can you see if you use it?

Reply:
Looks like this is where all the appsettings are defined. So adding it here. We can remove the one in Rpc constants and reference this file if needed.